xradar/io/backends/nexrad_level2.py

```diff
@@ -1631,6 +1637,8 @@ def open_nexradlevel2_datatree(
     )
     ls_ds: list[xr.Dataset] = [sweep_dict[sweep] for sweep in sweep_dict.keys()]
     ls_ds.insert(0, xr.Dataset())
+    if title:
+        ls_ds[1].attrs["title"] = title
```

Comment:
Is this by purpose adding the title to the first sweep dataset? Why?

Reply:
OK, If you want to mention this within the file metadata, this might go into the root attributes.

If there is nothing which would override title in the future, `title` is totally fine. (see https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#description-of-file-contents)

Reply:
When setting this up to the first dataset, it will be transferred to the root in the `_assign_root` function.  Therefore, it will only be for the global/root attributes. I was thinking of using `comment`  (Miscellaneous information about the data or methods used to produce it.); however, I think we already use it as "im/exported using xradar". Can we add additional information to the `comment` attribute?

Reply:
Great! I will then move it to the comment attribute